reference/library/src/morpheus-master/sass/modules/_toolmenu.scss

```diff
@@ -224,6 +229,7 @@ nav#subSites{
 				text-align: center;
 				text-decoration: none;
 				&:hover{
+					transform: scale(1.15);
```

Comment:
Hi, Could you make a failback for browsers which does not support a transform without vendor prefixes?